### PR TITLE
Implement page logging

### PR DESCRIPTION
**Pull request by jarrydlee**
We will need to add more info into the logs, but we'll wait for the queue to be fully developed before doing that. 

fixes https://github.com/PagerNation/PagerNation/issues/140 
fixes https://github.com/PagerNation/PagerNation/issues/61

### CODE REVIEW

File: server/services/alert.js [hunk 1]
```diff
@@ -12,10 +14,13 @@ function createAlert(ticket) {
     .then((user) => {
       const firstDevice = user.devices[0];
       sendPage(ticket, user, firstDevice);
-    });
+    })
+    .then(() => ticketService.addAction(ticket.id, actionTypes.CREATED));
 }
 
 function sendPage(ticket, user, device) {
+  ticketService.addAction(ticket.id, actionTypes.PAGE_SENT, user.id);
```
Comment (kalebdavis) [inline server/services/alert.js]: If this is a promise, does it need to be handled with a `then`?

File: server/services/alert.js [hunk 2]
```diff
@@ -12,10 +14,13 @@ function createAlert(ticket) {
     .then((user) => {
       const firstDevice = user.devices[0];
       sendPage(ticket, user, firstDevice);
-    });
+    })
+    .then(() => ticketService.addAction(ticket.id, actionTypes.CREATED));
```
Comment (kalebdavis) [inline server/services/alert.js]: The logging should be tested in an alert test

File: server/services/alert.js (hunk 1 above)
Comment (jarrydlee) [inline server/services/alert.js]: I'm not worried about it being in order, so it doesn't matter

File: server/services/alert.js (hunk 2 above)
Comment (jarrydlee) [inline server/services/alert.js]: I think once the `createAlert()` is refactored to work with the new queue it should be written. For right now I know it's about to change next week, so it seems pointless.

File: server/routes/alert.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import alertCtrl from '../controllers/alert';
+
+const router = express.Router(); // eslint-disable-line new-cap
+
+router.route('/')
+  .post(alertCtrl.sendPage);
```
Comment (kalebdavis) [inline server/routes/alert.js]: Is this allowing anyone to manually send a page?

File: server/services/ticket.js [hunk 4]
```diff
@@ -41,10 +42,35 @@ function getTicketsByDate(filterOpts) {
...
+    userId: Joi.string().hex().length(24),
+    device: Joi.object()
+  };
+  return JoiHelper.validate({ ticketId, actionType, userId, device }, schema)
+    .then(values =>
+      Ticket.addAction(values.ticketId, values.actionType, values.userId, values.device));
+}
+
+function removeAction(ticketId, actionType, timestamp, userId) {
+  const schema = {
```
Comment (kalebdavis) [inline server/services/ticket.js]: Why is this schema different than the `addAction` schema?

Comment (jarrydlee) [inline server/services/ticket.js]: Currently the action is removed by timestamp, actionType, and userId because those are always on the ticket. Thinking about it now, it makes more sense to add a UUID onto the actions. This also isn't actually on this PR, it's just moved from a different file, this change should be in a different PR.

File: server/routes/alert.js (hunk 3 above)
Comment (jarrydlee) [inline server/routes/alert.js]: I was thinking about this a little bit and it seems like we could create an admin user for the queue. Seems a little janky, but for now I think it's fine. What do you think?
